src/cls/rgw/cls_rgw_types.h

```diff
@@ -199,12 +199,14 @@ struct rgw_bucket_dir_entry_meta {
   std::string owner_display_name;
   std::string content_type;
   uint64_t accounted_size;
+  uint64_t size_rounded;
   std::string user_data;
   std::string storage_class;
   bool appendable;
 
   rgw_bucket_dir_entry_meta() :
-    category(RGWObjCategory::None), size(0), accounted_size(0), appendable(false) { }
+    category(RGWObjCategory::None), size(0), accounted_size(0),
+    size_rounded(0), appendable(false) { }
```

Comment:
We should test this with rados, it must not create any problem with rados as backend.

Reply:
sure will test it with plain ceph.

Reply:
@jjxsg : it's not about our compilation/build problem; it's about merge later when we merge into Ceph's main branch. I don't agree with such change.

Reply:
I agree, we should discuss it with ceph folks first. 
They are open to changes even in their core code as long as it does not affect their workflow and design principals. So even before discussion with ceph folks, we should check if it is breaking any other stores (rados, DB). If it is breaking, we and they as well will straightforward reject these changes.